package.json

```diff
@@ -51,26 +51,29 @@
     "uniqid": "^4.0.0"
   },
   "devDependencies": {
-    "babel-eslint": "^8.2.1",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-preset-power-assert": "^1.0.0",
-    "babel-preset-react": "^6.24.1",
-    "electron": "^1.7.11",
-    "electron-builder": "^19.55.3",
-    "electron-localshortcut": "^2.0.2",
-    "electron-webpack": "^1.12.0",
+    "@babel/core": "^7.1.2",
+    "@babel/plugin-syntax-dynamic-import": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-eslint": "^10.0.0",
+    "babel-loader": "^8.0.4",
+    "babel-preset-power-assert": "^3.0.0",
+    "electron": "^3.0.2",
+    "electron-builder": "^20.28.4",
+    "electron-localshortcut": "^3.0.4",
+    "electron-webpack": "^2.3.1",
     "eslint": "^4.16.0",
     "eslint-plugin-react": "^7.6.0",
-    "mocha": "^4.1.0",
-    "mocha-webpack": "^1.0.1",
+    "mocha": "^5.2.0",
+    "mocha-webpack": "^2.0.0-beta.0",
     "node-sass": "4.7.2",
     "power-assert": "^1.4.4",
     "redux-mock-store": "^1.4.0",
     "rimraf": "^2.6.2",
     "sass-loader": "^6.0.6",
     "sinon": "^4.2.1",
-    "webpack": "^3.10.0",
-    "webpack-sources": "1.0.1"
+    "webpack": "^4.20.2",
+    "webpack-sources": "^1.3.0"
```

Comment:
Did the `webpack-sources` shenanigans get fixed? I remember having to rollback and pin `1.0.1` because of an issue.

Reply:
https://github.com/james-proxy/james/pull/347

Guessing this must've been fixed since there's no build issues. Just FYI in case you forgot. 👍